dart/lib/src/http_client/failed_request_client.dart

```diff
@@ -166,7 +167,12 @@ class FailedRequestClient extends BaseClient {
       throwable: throwableMechanism,
       request: sentryRequest,
     );
-    await _hub.captureEvent(event, stackTrace: stackTrace);
+    await _hub.captureEvent(
+      event,
+      stackTrace: stackTrace,
+      hint: Hint.fromMap(
+          {'request': request, 'requestDuration': requestDuration}),
```

Comment:
I'd add `request` and `response` here.
The `response` object is available when you merge `main` into the `v7` branch, and v7 into this branch.